### ISSUE TITLE
Screen reader error in "Save and Process"

### ISSUE THREAD

**Issue by Arnei** [elan-ev/opencast-editor#268]
The button "Start processing" is read two times. Also the buttons are weirdly named. -Wien

**Comment by Arnei** [comment]
At least with orca screenreader on fedora, this is not the case (anymore?). I also lack ideas for non-weird yet still explanatory names, so I'll close this.
